Packages/Netherlands3D/Timeline/Runtime/Scripts/RectTransformExtensions.cs

```diff
@@ -0,0 +1,44 @@
+using UnityEngine;
+
+namespace SLIDDES.UI
+{
+    /// <summary>
+    /// Extension class for RectTransform
+    /// </summary>
+    public static class RectTransformExtensions
```

Comment:
Nice, erg handig! 
-,misschien dat deze beter bij de andere Extention scripts kan staan in de Core?
Core/Runtime/Scripts/<>

Reply:
Heb hem verplaast naar de core in de extention folder